pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -338,6 +337,13 @@ public BrokerResponse handleRequest(JsonNode request, @Nullable RequesterIdentit
     return brokerResponse;
   }
 
+  private PinotQueryRequest getPinotQueryRequest(JsonNode request) {
+    if (request.has(SQL)) {
```

Comment:
Would be good to assert that only one of SQL|PQL is specified in the request.

Reply:
Since sql support is newly added, I feel it's ok to keep sql parsing as first priority comparing to PQL.
So existing PQL user won't get affected.